docs/sources/tutorials/alerting-get-started/index.md

```diff
@@ -48,27 +62,57 @@ To demonstrate the observation of data using the Grafana stack, download the fil
 
 1. Clone the [tutorial environment repository](https://www.github.com/grafana/tutorial-environment).
 
+   <!-- INTERACTIVE exec START -->
    ```
    git clone https://github.com/grafana/tutorial-environment.git
    ```
+   <!-- INTERACTIVE exec END -->
 
 1. Change to the directory where you cloned the repository:
 
+   <!-- INTERACTIVE exec START -->
    ```
    cd tutorial-environment
    ```
+   <!-- INTERACTIVE exec END -->
 
 1. Run the Grafana stack:
 
+   <!-- INTERACTIVE ignore START -->
    ```
    docker compose up -d
    ```
+   <!-- INTERACTIVE ignore END -->
+
+   {{< docs/ignore >}}
+
+   <!-- INTERACTIVE exec START -->
+    ```bash 
+    docker-compose up -d 
+    ```
+   <!-- INTERACTIVE exec END -->
+
+   {{< /docs/ignore >}}
 
    The first time you run `docker compose up -d`, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
 
-   {{< admonition type="note" >}}
-   If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
-   {{< /admonition >}}
+   
+   **NOTE:** If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
```

Comment:
The admonition shortcode that you removed is the preferred way to format notes in our documentation.  I'm not sure where this tutorial ends up being published, so YMMV.

https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition

Reply:
Thanks for your review @JStickler 

The problem is that shortcode of type note, tip, etc.. can't be rendered in killercoda sandbox as of yet. I've added a ignore marker, so the shortcode is ignored in Killercoda while it still shows in grafana.com